idaes/power_generation/unit_models/waterpipe.py

```diff
@@ -391,7 +391,7 @@ def initialize(blk, state_args=None, outlvl=idaeslog.NOTSET,
             )
 
         # Unfix outlet enthalpy and pressure
-        for t in blk.flowsheet().config.time:
+        for t in blk.flowsheet().time:
             blk.control_volume.properties_out[t].pressure.unfix()
```

Comment:
This idiom shows up a lot, and could potentially be made simpler (in a future PR) with just
```python
blk.control_volume.properties_out[:].pressure.unfix()
```

Reply:
@jsiirola I didn't realize that would actually work for fixing and unfixing. However, that applies mostly to moved developers rather than core developers, so getting them to adopt that will take some work.

Reply:
It actually works for any method (as well as property setters), and expands to effectively be the loop you would have written.  It is potentially slightly slower, as it uses an `IndexedComponent_slice` iterator, which has some overhead relative to the simple loop.  BUT, the syntax is pretty.  ;)